chapters/planning-data-work.tex

```diff
@@ -365,11 +365,12 @@ \subsection{Organizing files and folder structures}
 and for the files that manage final analytical work.
 The command also has some flexibility for the addition of
 folders for non-primary data sources, although this is less well developed.
-The package also includes the \texttt{iegitaddmd} command,
-which can place a \texttt{README.md} file in each of these folders.
-These \textbf{Markdown} files provide an easy and Git-compatible way
+The \texttt{ietoolkit} package also includes the \texttt{iegitaddmd} command,
+which can place \texttt{README.md} placeholder files in your folders so that 
+your folder structure can be shared using git. Since these placeholder files are in
```

Comment:
Do we capitalize "git" when it is alone or no?

Reply:
You have done so so lets keep doing that